06-building-a-program-start-to-finish/setting-up-software-development-environments-(deployment)/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: Heroku
+- **Deliver**: two live URLs to your `staging` & `production` apps
```

Comment:
@q-ode Could also throw in the `review app` feature. There are arguments for and against whether or not `it's a nice to have`, but I strongly feel it's an important pro-tip for them to know the benefits off early on.